src/libcharon/plugins/kernel_super/kernel_super_ipsec.c

```diff
@@ -155,7 +161,7 @@ static status_t del_route_from_routing_table(private_kernel_super_ipsec_t* this,
 											 bool* is_route_installed);
 
 static void remove_ike_sa_matching(struct pm_terminator_conn* const terminator_conn);
-
```

Comment:
Keep 2 empty lines between the functions declarations section and the actual code.